### PR TITLE
Postal code added to IPv4.

### PR DESCRIPTION
**Pull request by workheap**
Very simple change to add zip code. Please consider.

### CODE REVIEW

File: lib/geoip.js [hunk 1]
```diff
@@ -107,7 +107,8 @@ function lookup4(ip) {
 				geodata.region = locBuffer.toString('utf8', (locId * locRecordSize) + 2, (locId * locRecordSize) + 4).replace(/\u0000.*/, '');
 				geodata.ll = [locBuffer.readInt32BE((locId * locRecordSize) + 4) / 10000, locBuffer.readInt32BE((locId * locRecordSize) + 8) / 10000];
 				geodata.metro = locBuffer.readInt32BE((locId * locRecordSize) + 12);
-				geodata.city = locBuffer.toString('utf8', (locId * locRecordSize) + 16, (locId * locRecordSize) + locRecordSize).replace(/\u0000.*/, '');
+				geodata.zip = locBuffer.readInt32BE((locId * locRecordSize) + 16);
```
Comment (bluesmoon) [inline lib/geoip.js]: I think you might also need to change `locRecordSize` to account for the change in record size.

Comment (bluesmoon) [inline lib/geoip.js]: NVM, it seems to be fixed at 64bytes.

**Comment by bluesmoon** [comment]
Can you also update the `README.md` file with docs?

File: lib/geoip.js (hunk 1 above)
Comment (workheap) [inline lib/geoip.js]: It's actually 64 bytes, I checked max city length, looks like it's sufficient.. Will update README.md for sure.

**Comment by bluesmoon** [comment]
@workheap I wonder if you can look into this, but zip codes that start with a 0 end up losing that 0 in the display.  Eg: MA & NY have ZIP codes that start with 02 and 01.

**Comment by simison** [comment]
@bluesmoon it's because zip is now handled as an integer while it should probably be string.

Does this library work only with US zip codes? Because e.g. [Canadian postal codes](https://en.wikipedia.org/wiki/Postal_codes_in_Canada) have letters in them.

**Comment by simison** [comment]
The code also mixes terms "postal code" and "zip". [ZIP kinda suggests](https://en.wikipedia.org/wiki/ZIP_Code) it's an US only feature...

**Comment by bluesmoon** [comment]
I agree, these should both be fixed.  @simison wanna take a stab at it?

**Comment by simison** [comment]
@bluesmoon nah sorry, I was looking for a library for this purpose so that's why I was looking trough issues/PRs here; to figure out health of this lib. (Looks good!)

But I'm not even sure if we'll implement the feature I'm thinking... :-)

It looks fairly simple since there are other string values already implemented and then just add a test maybe.

**Pull request merged**